### DISCUSSION TITLE
Is it possible to remove the filter part from the selection?

### DISCUSSION THREAD

**Discussion by Allyedge** [erikgeiser/promptkit#8] [Q&A] [answered]
Hey, I am using the list component in a project, but I couldn't really find a way to remove the filter section.

The reason why I don't want it is because the list is quite small, so it isn't really necessary.

How exactly can I do it?

**Comment by Allyedge** [comment]
> Hey, I am using the list component in a project, but I couldn't really find a way to remove the filter section.
> 
> The reason why I don't want it is because the list is quite small, so it isn't really necessary.
> 
> How exactly can I do it?

I realised that I said list and not selection.

I meant the selection component, my bad.

**Comment by erikgeiser** [comment, marked as answer]
Sorry for the late reply, I always forget to keep track of these Q&A questions. Of course you can remove the filter by removing the `selection.Filter` function like this:
```go
sp := selection.New("Pick one!", selection.Choices([]string{"a", "b", "c"}))
sp.Filter = nil
```
